canvas_modules/common-canvas/src/common-properties/controls/abstract-table.jsx

```diff
@@ -353,8 +357,12 @@ export default class AbstractTable extends React.Component {
 			}
 		} else { // defaults to inline control
 			tableInfo.editStyle = EditStyle.INLINE;
+			const factoryElem = ControlFactory.createControl(columnDefObj, propertyId, tableInfo);
+			if (selectSummaryRow && factoryElem.props.control.controlType === "oneofselect") {
+				factoryElem.props.control.clearAfterSelection = true;
+			}
```

Comment:
Code on line 360-363 can be removed. After latest code refactor, `clearAfterSelection` isn't used anywhere.